src/main/java/org/ibp/api/java/impl/middleware/common/validator/GermplasmListValidator.java

```diff
@@ -150,6 +154,17 @@ public void validateListIsUnlocked(final GermplasmList germplasmList) {
 		}
 	}
 
+	public void validateExceededMaximunColumnsAllowed(final GermplasmList germplasmList ,final List<GermplasmListDataUpdateViewDTO> view) {
+		final int entryDetailsLength = germplasmList.getView().stream().filter(germplasmListDataView -> germplasmListDataView.isEntryDetailColumn()).collect(
```

Comment:
@cuenyad I think we can do the same with variables to prevent this problem to be caused by entry variables. You can move the count search to middleware and sum both props and columns and make the proper validation in both services in BMSAPI